package.json

```diff
@@ -20,17 +20,17 @@
   "homepage": "https://github.com/cloudbees/blueocean-acceptance-test#readme",
   "devDependencies": {
     "@jenkins-cd/sse-gateway": "0.0.7",
```

Comment:
@tfennelly https://github.com/jenkinsci/blueocean-acceptance-test/pull/34/commits/3ff7588d12c049cc07bc8c5dbcbb818c26d3cd56 I did that but still failed